components/AccountWidget/AccountWidget.jsx

```diff
@@ -23,7 +23,7 @@ const AccountWidget = ({ loading, error, user, style }) => (
     {error && <Error />}
     {!error && !loading && user ? (
       <div>
-        <Link className={styles.tab} to={`/players/${user.account_id}`}>{"Profile"}</Link>
+        <Link className={styles.tab} to={`/players/${user.account_id}/overview`}>{"Profile"}</Link>
```

Comment:
users entering without "overview" should also get the same page.  I assume this is done to cause the inkbar to show up on the navbar (there was a previous issue with this)